Core/Pixel.swift

```diff
@@ -131,6 +131,8 @@ public enum PixelName: String {
     case feedbackNegativePerformanceVideo = "mfbs_negative_performance_video"
     case feedbackNegativePerformanceOther = "mfbs_negative_performance_other"
     
+    case etagStoreOOSWithDisconnectMe = "mdebug_etag_disconnect_oos"
```

Comment:
the top level pixel is `m` so they need to be in the format:
`m_whatever_your_pixel_is`

Personally I prefer the names to be a bit more abstract and brief, e.g. 
`m_dbg_dc_oos`
`m_dbg_el_oos`

But that's optional.

Reply:
Done, thanks for pointing that out :)